app.py

```diff
@@ -1,12 +1,25 @@
 """Angular+Flask AppEnginer Starter App"""
 import os
 
+import json
 import flask
 import flask_cors
 from dotenv import load_dotenv
+from rauth import OAuth2Service
```

Comment:
google style - [importing member of a module](https://google.github.io/styleguide/pyguide.html#22-imports)

should instead do:
`import rauth`

and then everywhere

`rauth.OAuth2Service`